Setup/UpgradeSchema.php

```diff
@@ -49,6 +49,35 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
                 );
         }
 
+        $swedbankPayQuoteTable = 'swedbank_pay_quotes';
+        $swedbankPayOrderTable = 'swedbank_pay_orders';
```

Comment:
Adjust the Codacy ruleset or shorten the name of this variable to less than 20 characters as suggested?

Reply:
I actually do not agree with shorter variable names. To me, it's better to have a long but meaningful variable name than a short one. We should adjust the codacy ruleset.